CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Note: For changes to the API, see https://shopify.dev/changelog?filter=api
 - [#1314](https://github.com/Shopify/shopify-api-ruby/pull/1314)
   - Add new session util method `SessionUtils::session_id_from_shopify_id_token`
   - `SessionUtils::current_session_id` now accepts shopify Id token in the format of `Bearer this_token` or just `this_token`
+- [#1315](https://github.com/Shopify/shopify-api-ruby/pull/1315) Add helper/alias methods to `ShopifyAPI::Auth::JwtPayload`
```

Comment:
Maybe we should list the methods here or just give an indication of what their purpose is?